<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodePropertiesWindow</class>
 <widget class="QDialog" name="NodePropertiesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Node properties</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>781</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="sourceLayout">
    <item>
     <widget class="QLabel" name="sourceLabel">
      <property name="minimumSize">
       <size>
        <width>779</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>779</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Source code</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="source">
      <property name="font">
       <font>
        <family>Courier New</family>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="inLayout">
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="inListLabel">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>IN</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="inList"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>781</width>
     <height>86</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="outLayout">
    <item alignment="Qt::AlignLeft">
     <widget class="QLabel" name="outListLabel">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>OUT</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="outList"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>781</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonLayout">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
